### PR TITLE
Upgrade CubeMX Version to 6.2.0

### PR DESCRIPTION
**Pull request by jonathanlew**
### Description
The [STM32CubeMX link for 5.6.1](https://www.st.com/content/ccc/resource/technical/software/sw_development_suite/group0/01/9e/e3/0c/3d/ff/41/7b/stm32cubemx_v5-6-1/files/stm32cubemx_v5-6-1.zip/jcr:content/translations/en.stm32cubemx_v5-6-1.zip) doesn't work anymore. Upgrading to 6.2.0 would fix this issue.

### Testing Done
CI, requesting @akhilveeraghanta firmware testing

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [ ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: src/firmware_new/boards/frankie_v1/frankie_v1.ioc [hunk 1]
```diff
@@ -1,635 +1,639 @@
...
-RCC.PLLCLKFreq_Value=216000000
-PB14.GPIO_Label=LD3 [Red]
-PG6.GPIO_Label=USB_PowerSwitchOn [STMPS2151STR_EN]
-RCC.PLLQCLKFreq_Value=48000000
-PD3.GPIOParameters=GPIO_Label
-LWIP.LWIP_IPV6_DUP_DETECT_ATTEMPTS=3
-PD8.Locked=true
-RCC.RTCFreq_Value=32000
-PA3.GPIOParameters=GPIO_Label
+CORTEX_M7.AccessPermission-Cortex_Memory_Protection_Unit_Region0_Settings=MPU_REGION_FULL_ACCESS
```
Comment (jonathanlew) [inline src/firmware_new/boards/frankie_v1/frankie_v1.ioc:2]: :warning: I opened this ioc in STM32CubeMX 6.2.0 and clicked migrate

**Review by akhilveeraghanta — APPROVED** [review]
lgtm and works on my stm32

File: src/firmware_new/boards/frankie_v1/frankie_v1.ioc (hunk 1 above)
Comment (akhilveeraghanta) [reply src/firmware_new/boards/frankie_v1/frankie_v1.ioc:2]: that seemed to actually work surprisingly well, version 6.2.0 codegen seems to be better, none of the code in user sections got wiped

**Review by kelvinkoon — APPROVED** [review]
LGTM from Akhil's confirmation

**Pull request merged**